vitest.config.js

```diff
@@ -0,0 +1,38 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    browser: {
+      provider: 'playwright',
+      enabled: true,
+      headless: true,
+      screenshotFailures: false,
+      instances: [{ browser: 'chromium' }],
+    },
+    globals: true,
+
+    // CSS bundle relied upon by accessibility tests (eg. for color-contrast
+    // checks).
+    setupFiles: './build/styles/test.css',
+    include: [
+      // Test bundle
+      './build/scripts/tests.bundle.js',
+    ],
+
+    coverage: {
+      enabled: true,
+      provider: 'istanbul',
+      reportsDirectory: './coverage',
+      reporter: ['json', 'clover', 'html'],
```

Comment:
What is the `clover` reporter? Do we need it?

Reply:
Isn't clover the one used by codecov?

Reply:
Oh, they support the json one as well. I'll remove clover then.